script.js

```diff
@@ -1,6 +1,244 @@
+    if (working){
+        hoursLeft = workHourTotal;
+        minutesLeft = workMinuteTotal;
+    }
+    else{
+        hoursLeft = breakHourTotal;
+        minutesLeft = breakMinuteTotal;
+    }
+
+    // every second
+    let second = setInterval(function(){countdown()}, 1000);
+    // when timer runs out
+    
+}
+function countdown(){
+    if (secondsLeft===0){
+        if (minutesLeft===0){
+            if (hoursLeft===0){
+                console.log("Time is up!");
+            }
+            else{
+                hoursLeft -=1;
+                minutesLeft = 60;
+                secondsLeft = 60;
+            }
+        }
+        else{
+            minutesLeft -= 1;
+            secondsLeft = 60;
+        }
+    }
+    else{
+        secondsLeft -= 1;
+    }
+    timerHour.textContent = `${hoursLeft}:`;
+    timerMinute.textContent =`${minutesLeft}:`;
+    timerSecond.textContent = secondsLeft;
+    if (hoursLeft===0&&minutesLeft===0&&secondsLeft===0){
+        console.log("timer has run out");
+        if (working){
+            alert("Your work time is over! Time to take a break!");
+            working = false;
+            pickActivity()
+        }
+        else{
+            alert("Your break is over! Get to work!");
+            working=true;
+            workOrBreak.textContent = "You are working right now!";
+        }
+        // makeTimer()
+        if (working){
+            timerHour.textContent = `${String(workHourTotal)}:`;
+            timerMinute.textContent = `${String(workMinuteTotal)}:`;
+        }
+        else{
+            timerHour.textContent = `${String(breakHourTotal)}:`;
+            timerMinute.textContent = `${String(breakMinuteTotal)}:`;
+        }
+        timerSecond.textContent = secondsTotal;
+        if (working){
+            hoursLeft = workHourTotal;
+            minutesLeft = workMinuteTotal;
+        }
+        else{
+            hoursLeft = breakHourTotal;
+            minutesLeft = breakMinuteTotal;
+        }
+    }
+}
+function pickActivity (){
+    let activities
+    if (breakHourTotal>=1){
+        activities = longActivities
+    }
+    else{
+        activities = shortActivities
+    }
+    // activities = shortActivities
+    let x = Math.random()*activities.length
+    let i = Math.floor(x);
+    let activity = activities[i];
+    console.log(i)
+    console.log(activity)
+    console.log(activities)
+    workOrBreak.textContent = `You are on break right now! You should ${activity} while you're taking a break!`
+}
+async function getActivity(){
+    console.log("getting activity");
+    const response = await fetch ("activities.json");
+    const activities = await response.json();
+    console.log(activities)
+    for (let currentActivity of activities){
+        if (currentActivity.time==="long"){
+            longActivities.push(currentActivity.name)
+        }
+        if (currentActivity.time==="short"){
+            shortActivities.push(currentActivity.name)
+        }
+    }
```

Comment:
This was a cool way to split up your json into different files. You could have just hardcoded this into the json, something like: 

```json
{
   shortActivities: ["draw something", "make origami"],
   long Activities: ["example 1" "example 2"],
}
```

Just a thought!